RUFAS/routines/animal/animal_module_reporter.py

```diff
@@ -238,7 +238,7 @@ def report_ration_interval_data(cls, pen_list: List[Pen], feed: Feed, simulation
             AnimalModuleReporter.data_padder(
                 f"{classname}.{funcname}.ration_nutrient_amount_pen_0_CALF",
                 f"{classname}.{funcname}.ration_nutrient_amount_pen_{pen.id}_{pen.animal_combination.name}",
-                {},
+                {variable: 0 for variable in nutrient_amount.keys()},
```

Comment:
```suggestion
                {key: val for key, val in nutrient_amount.items()},
```
I think this would be more correct than adding in 0. Same goes for all the other instances of adding in values.

Reply:
I don't think this is correct because we want to pad the missing days with 0s when pens are created on the fly.